packages/components/slider/src/composables/use-slide.ts

```diff
@@ -19,9 +19,9 @@ export const useSlide = (
 
   const slider = shallowRef<HTMLElement>()
 
-  const firstButton = ref<SliderButtonInstance>()
+  const firstButton = ref()
```

Comment:
Why do you do this?

Reply:
sorry, if I don't do this. when run `pnpm build`, it will show an error like this
![image](https://github.com/user-attachments/assets/d3953eb0-d7a4-4eff-b4e8-10485331bb68)

today, I pushed again, I found another way to solve this error, you can review it, thanks